docs/setup-guide/scale-out-local-dev-setup.md

```diff
@@ -59,12 +67,33 @@ Note:
 ```
 export IS_RESOURCE_PARTITION=true
 export TENANT_SERVER=[TP1_IP]<,[TP2_IP]>
+export RESOURCE_PARTITION_POD_CIDR=pod-cidr
 ```
 
+an examplative allocation of pod cidr for 2 RPs could be
```

Comment:
the setup should also describe the kube-proxy's  --cluster-cidr parmaeter should be the super set of the RP cluster IPs, in this case, should be 10.0.0.0/8 if assuming all RPs will be 10.x.0.0/16 based.

Reply:
thanks for the directions. Haproxy seems have no place for cluster-cidr (of service IP ranges); it only cares about TP/RP IP addresses. Seems non issue.

Reply:
--cluster-cidr is not mandatory in our cluster; neither scale-up nor scale-out script provide value of cluster-cidr for kube-proxy.  it should be fine not specifying it.

Reply:
this is used to determine the internal and external traffics, even it is not required, but from this code we should have it specified and supported.

```
if len(clusterCIDR) == 0 {
		klog.Warning("clusterCIDR not specified, unable to distinguish between internal and external traffic")
	} else if utilnet.IsIPv6CIDRString(clusterCIDR) != ipt.IsIpv6() {
		return nil, fmt.Errorf("clusterCIDR %s has incorrect IP version: expect isIPv6=%t", clusterCIDR, ipt.IsIpv6())
	}
```

Reply:
my point is we did not specify it in the first place with arktos-up.sh. This is not 2x2 scale-out specific issue; better to cope it out side of scale-out service support PRs. I'll create an issue to track it.

Reply:
that might be OK for now. my original thought this can be just a superset of RP cluster-cidrs. a issue to track this is good for now.

Reply:
issue @1201 was created to track this discussion/concern